site/_posts/2018-12-12-reinventing-the-proxy.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Reinventing the proxy for our remote consoles"
+author: skateman
+date: 2018-12-12
+comments: true
+published: true
+tags: purr
```

Comment:
we use general tags for this blog, please consider using `developers` and/or `ruby`